Updated README, documentation assets, and content (#1028)
liveblocks/liveblocks: docs/pages/products/storage.mdx

    it in real‑time."
---

> Storage is a persisted data store optimized for real-time collaborative
> documents. Store your document’s data with Storage to enable people to edit
> together it in real‑time. `LiveList`, `LiveMap`, and `LiveObject` conflict-free
> data types can be stored to build all sorts of multiplayer editors: forms,
> creative tools, text, and more.

Reviewer: ```suggestion
Storage is a persisted data store optimized for real-time collaborative
documents, allowing people to edit data together in real‑time. `LiveList`, 
`LiveMap`, and `LiveObject` conflict-free data types can be stored to 
build all sorts of multiplayer editors: forms, creative tools, text, and 
more.
```
I think this sounds a bit better
